README.md

```diff
@@ -28,14 +34,24 @@ Find another group to work with. The other group should be students who neither
 
 ***Q1.2 Share some notes from your interview here and try to describe the "need" with a very brief description:***
 
-(6 minutes) Reverse roles. The Customers are now Designers and vice versa.  Repeat the interview process. (Designers, complete the question above in that space)
+Cell phones have a lot of radiation, goal: to create a concept that gives health benefits to user or
```

Comment:
Intriguing idea, I know the long term effects of being exposed to radiation from phones has been something studied frequently in today's age.